.release/security-scan.hcl

```
@@ -9,8 +9,18 @@ container {
 
 binary {
 	secrets      = false
-	go_modules   = false
+	go_modules   = true
 	osv          = true
 	oss_index    = true
 	nvd          = false
 }
+
+# Triage items that are _safe_ to ignore here. Note that this list should be
+# periodically cleaned up to remove items that are no longer found by the scanner.
+triage {
+	suppress {
+		vulnerabilities = [
+			"GO-2022-0635", // github.com/aws/aws-sdk-go@v1.55.5
+		]
+	}
+}
```

I think the `triage` block needs to be inside the `binary` block ([docs](https://github.com/hashicorp/security-scanner/blob/main/CONFIG.md#binary)), otherwise the scan throws an error:

```
Error: failed to decode scan config contents, unable to gracefully handle: /Users/bishop/projects/vault-review/.release/security-scan.hcl:20,1-7: Unsupported block type; Blocks of type "triage" are not expected here.
failed to decode scan config contents, unable to gracefully handle: /Users/bishop/projects/vault-review/.release/security-scan.hcl:20,1-7: Unsupported block type; Blocks of type "triage" are not expected here.
```

It's probably a good idea to enable go_stdlib scanning too:

```suggestion
	go_modules   = true
	go_stdlib    = true
	osv          = true
	oss_index    = true
	nvd          = false

	# Triage items that are _safe_ to ignore here. Note that this list should be
	# periodically cleaned up to remove items that are no longer found by the scanner.
	triage {
		suppress {
			vulnerabilities = [
				"GO-2022-0635", // github.com/aws/aws-sdk-go@v1.55.5
			]
		}
	}
}
```